src/binary/ibc-setup/commands/balances.spec.ts

```diff
@@ -0,0 +1,136 @@
+import fs from 'fs';
+import os from 'os';
+
+import test from 'ava';
+import sinon from 'sinon';
+
+import { IbcClient } from '../../../lib/ibcclient';
+import { generateMnemonic } from '../utils/generate-mnemonic';
+
+import { Options, run } from './balances';
+
+const consoleLog = sinon.stub(console, 'log');
+const fsReadFileSync = sinon.stub(fs, 'readFileSync');
+const ibcClient = sinon.stub(IbcClient, 'connectWithSigner');
+const mnemonic = generateMnemonic();
+
+function fakeBalance(amount: string) {
```

Comment:
Naming: `createFakeIbcClient`? Also if you make this an `async` function I don't think you need the `Promise.resolve` and the cast can be simpler.

Reply:
done in 3ce03b0